### DISCUSSION TITLE
Are polyfills-dom and polyfills-core-js files that babel created? How to config that?

### DISCUSSION THREAD

**Discussion by XueMeijing** [babel/babel#14141] [Q&A]
I'm using create-react-app and ionic which is an UI frame include webcomponents, so ,when i packaged the  polyfills-dom and polyfills-core-js files added in build folder. Now i want to delete this files，or let index.html import polyfills-dom directly, like polyfills-core-js do, or combine polyfills-dom into polyfills-core-js. Any suggestions? Not English speaker,sry for my English. Here are some screenshots
![image](https://user-images.githubusercontent.com/35559153/149105384-94a081a9-52f3-4b09-96d3-aa020d4f912f.png)
![image](https://user-images.githubusercontent.com/35559153/149107525-44df6102-a81d-43b4-ad5c-63d3a2083a92.png)

**Comment by nicolo-ribaudo** [comment]
Those files are not created by Babel. Maybe you are using a webpack plugin that creates them?

**Reply by XueMeijing** [reply]
> Those files are not created by Babel. Maybe you are using a webpack plugin that creates them?

Thanks for ur reply and suggestions, I'll have a try. Have a good day~
